### ISSUE TITLE
Opening and saving jpeg using CMYK color space drastically changes image's color

### ISSUE THREAD

**Issue by gsfreema** [SixLabors/ImageSharp#1238]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of ImageSharp
- [x] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
- [x] I have searched [open](https://github.com/SixLabors/ImageSharp/issues) and [closed](https://github.com/SixLabors/ImageSharp/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
The following image was created using the CMYK color space.

![SolidColor_CYMK](https://user-images.githubusercontent.com/5770230/85042859-b54ab800-b159-11ea-8a02-93c19beb18ad.jpeg)

Loading this file and then saving results in this image.

![Modified - SolidColor_CYMK](https://user-images.githubusercontent.com/5770230/85042550-56853e80-b159-11ea-8cc2-0f9d5389d820.jpeg)

(On my machine, the two images look the same in this issue but if you download the images, you will see the difference. Maybe github is experiencing the same thing 😃)

### Steps to Reproduce
The following code is all you need to convert the first image into the second one. Note that this is a .Net Core 3.1 app using C#8.
```cs
const string FileName = "SolidColor_CYMK.jpeg";
using var image = Image.Load(FileName);
image.Save($"Modified - {FileName}");
```

### System Configuration

- ImageSharp version: 1.0.0-rc.2
- Other ImageSharp packages and versions: N/A
- Environment (Operating system, version and so on): Win10 1809 x64
- .NET Framework version: .Net Core 3.1
- Additional information:

**Comment by JimBobSquarePants** [comment]
Yeah, that's expected. We don't have methods to correctly translate the CMYK color information. As you can see, many browsers do the same.

See previous related issues
https://github.com/SixLabors/ImageSharp/issues/129
https://github.com/SixLabors/ImageSharp/issues/74

And related Work-In-Progress PR.
https://github.com/SixLabors/ImageSharp/pull/273

**Comment by gsfreema** [comment]
Thanks for the quick response. I searched for CMYK and I had seen the other issue about ICC Profiles but I wasn't sure it was the same given that I thought I selected to not save the ICC profile when generating the image. I thought I limited it to just using CMYK and was semi-proud of myself since I am far from experienced with image processing 😄 .

**Comment by JimBobSquarePants** [comment]
Be proud. Image processing is hard! Hopefully we'll make it a little bit easier for all.

**Comment by ValZapod** [comment]
> Maybe github is experiencing the same thing

No, it does not. It does not alter the bitstream of images. If it is, this is a bug in Chrome.

The fist file is CMYK file because exiftool says: 

APP14 Flags 0                   : (none)
APP14 Flags 1                   : (none)
Color Transform                 : **Unknown (RGB or CMYK)**

This is ColorTransform = 0 which means it is RGB if 3 components and CMYK if 4. 
While second file outright says it is 

Y Cb Cr Sub Sampling            : YCbCr4:2:0 (2 2)
Image Size                      : 100x100
Megapixels                      : 0.010

So it is NOT CMYK AT ALL.

The first CMYK file does not have an ICC profile, since only RGB has a default of sRGB, it will be shown differently everywhere.

**Comment by timdinhdotcom** [comment]
Is there a way to convert an RGB image to CMYK?

**Comment by JimBobSquarePants** [comment]
We currently don't support encoding jpeg as CMYK so any conversion we do would be lost.

**Comment by JimBobSquarePants** [comment]
Fixed via #2120
